package.json

```diff
@@ -27,9 +27,11 @@
     "webpack": "^4.41.2"
   },
   "dependencies": {
+    "@types/lodash": "4.14.149",
```

Comment:
`@types` module should go under `devDependencies`. Can you move this one and the puppeteer one as well.